app/Providers/AppServiceProvider.php

```diff
@@ -44,6 +45,11 @@ public function boot()
      */
     public function register()
     {
-        //
+        $this->app->singleton(LoggerInterface::class, function(){
```

Comment:
Bind a monolog instance to the logger interface. Anywhere the LoggerInterface is type-hinted in the constructor of something, the dependency injection will inject in this monolog instance. Nice and easy.